src/Sarif.Multitool/MergeCommand.cs

```diff
@@ -32,14 +32,20 @@ public static int Run(MergeOptions mergeOptions)
                     combinedLog = new InsertOptionalDataVisitor(dataToInsert).VisitSarifLog(combinedLog);
                 }
 
+                string outputDirectory = mergeOptions.OutputFolderPath ?? Environment.CurrentDirectory;
```

Comment:
>outputDirectory [](start = 23, length = 15)

I see that this is a bug fix. Before it `mergeOptions.OutputFolderPath` was missing, you'd crash with a nullref on the next line.